src/lib.rs

```diff
@@ -151,6 +147,12 @@ pub mod errors {
                 description("Reserved Filename")
                 display("{} is reserved for internal use", filename.display())
             }
+
+            /// Error with a TOML file.
+            TomlQueryError(inner: ::toml_query::error::Error) {
```

Comment:
We don't need leading :: before crate name in 2018 edition.

Reply:
Oh, yea, I was just copy/pasting, and keeping the style. I noticed all the leading `::` and thought it was odd. I'll update after #944.